packages/perseus-editor/src/article-editor.tsx

```diff
@@ -417,6 +436,15 @@ export default class ArticleEditor extends React.Component<Props, State> {
     render(): React.ReactNode {
         return (
             <div className="framework-perseus perseus-article-editor">
+                {this.state.conversionWarningRequired && (
+                    <div style={{marginBottom: 10}}>
+                        <Banner
+                            text="Pre-existing Input Number Widgets have been converted to Numeric Inputs. Please review the changes before publishing."
```

Comment:
**nit**: "Pre-existing" threw me... where did they exist before? :) 

Perhaps "Deprecated Input Number widgets were found in this article! They have been automatically upgraded to Numeric Inputs. Please review the changes before publishing."